### PR TITLE
[16.0][IMP]helpdesk_mgmt: better attachment support

### PR DESCRIPTION
**Pull request by PicchiSeba**
Forward ports https://github.com/OCA/helpdesk/pull/463 which itself forward-ported the commit https://github.com/OCA/helpdesk/commit/656c2d933586fcb08a50488ee5b0cd27b56dfe03 from 12.0 to 14.0

### CODE REVIEW

**Review by flotho — APPROVED** [review]
LGTM,
already tested on production

File: helpdesk_mgmt/views/helpdesk_ticket_templates.xml [hunk 1]
```diff
@@ -483,16 +483,14 @@
                         for="attachment"
                     >Add Attachments</label>
                     <div class="col-md-7 col-sm-8">
-                        <div class="btn btn-default btn-file col-md-12">
```
Comment (pedrobaeza) [inline helpdesk_mgmt/views/helpdesk_ticket_templates.xml]: Why removing the div?

Comment (PicchiSeba) [reply helpdesk_mgmt/views/helpdesk_ticket_templates.xml]: A mistake on my part, I'm restoring it now

**Review by HekkiMelody — APPROVED** [review]
Code review, LGTM

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Comment by pedrobaeza** [comment]
To fw-port to upper versions.

**Pull request merged**